### PR TITLE
DOCS: reworded the "Linux desktop issues" paragraph from the manual

### PR DESCRIPTION
**Pull request by piorrro33**
The idle-inhibit Wayland protocol has been implemented into GNOME's GTK (with commit [5af7d6bf](https://gitlab.gnome.org/GNOME/gtk/-/commit/5af7d6bff3d2920aca118a36f13b23cbb68c1641)).

### CODE REVIEW

**Comment by Dudemanguy** [comment]
Unfortunately, mutter needs to implement this not just gtk (PR still pending [here](https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/111)) so the issue still exists. If you want to reword this to be slightly nice to GNOME I wouldn't be opposed but, as is, the rant is mostly warranted IMO.

**Comment by 07416** [comment]
> The idle-inhibit Wayland protocol has been implemented into GNOME's GTK (with commit `5af7d6bf`).

Can you edit the link in?

**Comment by piorrro33** [comment]
> Can you edit the link in?

Done :)
